src/databricks/labs/ucx/install.py

```diff
@@ -573,6 +593,14 @@ def _trigger_workflow(self, step: str):
         if self._prompts.confirm(f"Open {step} Job url that just triggered ? {job_url}"):
             webbrowser.open(job_url)
 
+    def _validate_config(self):
```

Comment:
Should this be moved to another place or does it makes sense to only validate the config as part of an existing installation?